000-local-env.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
Let's move this file to `vip-helpers/vip-local.php` and conditionally include it from `000-vip-init.php`.

I think we should still use `WPCOM_IS_VIP_ENV` for this check, and let `VIP_GO_ENV` track the 'environment name', rather than use it to determine where the code is running. 

Currently `VIP_GO_ENV` is set to `false` in dev, but IMO it should be set to something like `local-development`. The environment name constant specifies which 'version' of the site the code is running on, vs. _where_ the code is physically running (Go vs local).